### DISCUSSION TITLE
add一个完整的pdf文档提取方案

### DISCUSSION THREAD

**Discussion by Kyrie666** [RapidAI/RapidOCRPDF#19] [Ideas]
你好，我想提供一个更加完整的pdf提取方案，包括文字和表格等内容。总体是对其他几个库的一个方案整合，因为我目前刚好使用了您的其他库做pdf的提取。

**Comment by SWHL** [comment]
可以啊，我们也正有此意的。你可以直接在这个仓库提PR。我来跟进整合。
我在做一个版面还原的，也打算是这个，还没写完。可以看看：[RapidLayoutRecover](https://github.com/RapidAI/RapidLayoutRecover)

**Reply by Kyrie666** [reply]
好的，我之前主要是在操作提取pdf中表格，今天想实现完整的方案，确实发现layout提取的精度不够，造成整体效果不佳

**Reply by SWHL** [reply]
版面检测是整个流程的关键，期待加入进来，一起搞这块。
目前我们这里有小伙伴在搞表格识别了，版面分析接下来搞，我这里搞版面还原
